Gemfile.lock

```diff
@@ -1,9 +1,9 @@
 GIT
   remote: https://github.com/wordpress-mobile/release-toolkit
-  revision: 348de22c625922cca9a5060261fa7a0c4005786e
-  tag: 0.16.0
+  revision: 4efe6cd1585db9bbf8f3f0cd889037ceade8de6e
+  tag: 0.15.0
   specs:
-    fastlane-plugin-wpmreleasetoolkit (0.16.0)
+    fastlane-plugin-wpmreleasetoolkit (0.15.0)
```

Comment:
@JavonDavis It looks like this commit accidentally changed these to an earlier version: b565b4c929fd1ee0d49ed6a290ae1f30724264b6

Reply:
Thanks for catching that, this should be actually be 0.17

Reply:
This looks like it happened because I didn't do a bundle install with the new version because I had done a quick switch back to the old version for testing